src/pages/search.tsx

```diff
@@ -55,74 +60,71 @@ const Search = ({
         </div>
       </PageHeader>
       <GridContainer>
-        <>
-          <div className={styles.pageTitle}>
-            <h2>
-              {resultString} for ‘{query}’
-            </h2>
-          </div>
+        <div className={styles.pageTitle}>
+          <h2>
+            {resultString} for ‘{query}’
+          </h2>
+        </div>
+
+        <Grid row gap="lg">
+          <Grid col="auto">
+            {flags?.searchPageFilter ? (
+              <SearchFilter labels={labels} />
+            ) : (
+              <EPubsCard query={query} />
```

Comment:
Question: Are we removing the EPubsCard as part of the story?

Reply:
I have the same question. I keep forgetting to ask, but will bring it up!

Reply:
ePubs card should still be there

Reply:
@jbecker01 ok cool, that's an easy fix. Above or below the filter?